source/installation.rst

```diff
@@ -1,43 +1,125 @@
  .. _installation:
 
+==============================
 Installation & Getting Started
 ==============================
 
-NanoVer consists of two components, the python-based libraries
-for running simulations, nanover-server, and the Unity3D libraries
-and applications for visualising and interacting with simulations
+NanoVer consists of two components: the python-based libraries
+for running simulations, and the Unity3D libraries
+and applications for visualising & interacting with simulations
 in VR.
 
-* Install the server software (see below)
-* Interact with molecules in VR by `downloading the VR client <https://github.com/IRL2/nanover-imd/releases/download/nightly/StandaloneWindows64.zip>`_
-* Check out the :ref:`tutorials <Tutorials>`
+.. contents:: Contents
+    :depth: 3
 
+~~~~
+
+#######################
+User Installation Guide
+#######################
+
+.. _creating_conda_env:
+
+Creating a conda environment
+############################
+
+#. Install Conda through whichever program you prefer, e.g. `Miniforge <https://github.com/conda-forge/miniforge>`_.
+#. Open a terminal that you have conda installed in:
+
+    * On Windows, this is Windows Powershell.
+    * On Mac and Linux, this is the terminal.
+
+#. Create a conda environment (here we call the environment "nanover"):
+
+    .. code:: bash
+
+        conda create -n nanover "python>3.11"
+
+~~~~
+
+Installing the server
 #####################
-Installing the Server
-#####################
 
-Conda Installation
-##################
+If you have not already set up a NanoVer conda environment, please refer to :ref:`creating_conda_env`.
+
+#. Activate your NanoVer conda environment: ``conda activate nanover``
+
+#. Install the NanoVer packages: ``conda install -c irl -c conda-forge nanover-server``
+
+For information on how to run NanoVer servers, check out the :ref:`tutorials <Tutorials>`.
+
+~~~~
+
+Setting up the iMD-VR client
+############################
+
+There are two ways to set up the iMD-VR client:
+
+* conda installation of the NanoVer iMD package
+* downloading the latest release of NanoVer iMD (necessary to facilitate running natively on
+  VR headsets)
+
+**Conda installation**
+
+If you have not already set up a NanoVer conda environment, please refer to :ref:`creating_conda_env`.
```

Comment:
Is it ok that this is mentioned twice? Once when setting up the server and again when setting up the client.